src/lib/curl/base.c

```diff
@@ -84,3 +90,125 @@ void fr_curl_free(void)
 
 	curl_global_cleanup();
 }
+
+int fr_curl_easy_tls_init (fr_curl_io_request_t *randle, fr_curl_tls_t *conf)
+{
+	REQUEST *request = randle->request;
```

Comment:
Should be whitespace between declarations and function body, just add a return here.